azkaban-web-server/src/main/java/azkaban/webapp/servlet/NoteServlet.java

```diff
@@ -0,0 +1,125 @@
+/*
+ * Copyright 2017 LinkedIn Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ *
+ */
+package azkaban.webapp.servlet;
+
+import azkaban.server.session.Session;
+import azkaban.trigger.TriggerManagerException;
+import azkaban.user.Permission;
+import azkaban.user.Role;
+import azkaban.user.User;
+import azkaban.webapp.AzkabanWebServer;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+import javax.servlet.ServletConfig;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import org.apache.log4j.Logger;
+
+public class NoteServlet extends LoginAbstractAzkabanServlet {
+
+  private static final long serialVersionUID = 1L;
+  private static final Logger logger = Logger .getLogger(NoteServlet.class);
+
+  public static String type = null;
+  public static String message = null;
+  public static String url= null;
+  private AzkabanWebServer server;
+
+  @Override
+  public void init(final ServletConfig config) throws ServletException {
+    super.init(config);
+    this.server = (AzkabanWebServer) getApplication();
+  }
+
+  @Override
+  protected void handleGet(final HttpServletRequest req, final HttpServletResponse resp,
+      final Session session) throws ServletException, IOException {
+    if(isAdmin(session.getUser())) {
```

Comment:
What do you think of adding some errorMsg in the page if a user without ADMIN permission tries to access this page?